Pear/SceneDelegate.swift

```diff
@@ -92,3 +94,47 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
     }
 
 }
+
+extension SceneDelegate: UNUserNotificationCenterDelegate {
+
+    private func setupLocalNotifications() {
+        let center = UNUserNotificationCenter.current()
+        center.requestAuthorization(options: [.alert, .sound]) { (granted, error) in
+            print("granted: \(granted)")
```

Comment:
You should not prompt the permissions system dialog right when the app first opens. Apple says it's bad UX. You should first present an alert or some text that explains what you need notifications for, ask them if they're cool with it, and only if yes then present the system alert. If they say no, you should tell them they can always change their mind in settings.